### ISSUE TITLE
Regarding the relevant training parameter settings for LLaRA (Caser) and LLaRA (GRU4Rec).

### ISSUE THREAD

**Issue by wuren519** [ljy0ustc/LLaRA#19]
Dear Author,
        I have successfully replicated your LLaRA (SASRec) on three datasets, and my results are generally consistent with yours. I am now trying to replicate LLaRA (Caser) and LLaRA(GRU),However, the results generated by SASRec and Caser are the same. Therefore, I believe the PT files for Caser and GRU are missing under the three datasets. Currently, only the PT files for SASRec under the three datasets are provided. I hope the author can provide the missing files. Thank you to the author.
        
![image](https://github.com/user-attachments/assets/8ed92c59-c99e-4802-b2c4-68ca8bcc6f75)

**Comment by betoobusy** [comment]
> Dear Author, I have successfully replicated your LLaRA (SASRec) on three datasets, and my results are generally consistent with yours. I am now trying to replicate LLaRA (Caser) and LLaRA(GRU),However, the results generated by SASRec and Caser are the same. Therefore, I believe the PT files for Caser and GRU are missing under the three datasets. Currently, only the PT files for SASRec under the three datasets are provided. I hope the author can provide the missing files. Thank you to the author.
> 
> ![image](https://github.com/user-attachments/assets/8ed92c59-c99e-4802-b2c4-68ca8bcc6f75)

Hello, I have also noticed this issue. Have you solved it yet

**Comment by cl-liamchiu** [comment]
> Dear Author, I have successfully replicated your LLaRA (SASRec) on three datasets, and my results are generally consistent with yours. I am now trying to replicate LLaRA (Caser) and LLaRA(GRU),However, the results generated by SASRec and Caser are the same. Therefore, I believe the PT files for Caser and GRU are missing under the three datasets. Currently, only the PT files for SASRec under the three datasets are provided. I hope the author can provide the missing files. Thank you to the author.
> 
> ![image](https://github.com/user-attachments/assets/8ed92c59-c99e-4802-b2c4-68ca8bcc6f75)

Hello, may I ask for details about your training setup with LLaRA (SASRec), such as the type of GPU, batch size, and accumulate_grad_batches? I'm trying to reproduce the LLaRA (SASRec) results using an A6000 GPU with batch_size=4 and accumulate_grad_batches=32, but my results are as follows:

- metric = 0.2632
- test_hr = 0.3906
- test_prediction_valid = 0.6737

I would really appreciate any insights or suggestions on what might be causing the discrepancy. Thanks in advance!

**Comment by 13769846552** [comment]
Dear Author,
I have successfully replicated your LLaRA (SASRec) on three datasets, and my results are generally consistent with yours. I am now trying to replicate LLaRA (Caser) and LLaRA(GRU),However, the results generated by SASRec and Caser are the same. Therefore, I believe the PT files for Caser and GRU are missing under the three datasets. Currently, only the PT files for SASRec under the three datasets are provided. I hope the author can provide the missing files. Thank you to the author.
